.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Platform Information (please complete the following information):**
+ - OS (Including Linux Distribution): 
+ - Python Version: 
```

Comment:
Provide a code snippet for the user to capture this info, e.g. `python -c 'print(.....)'`?